labs/hlf-operator.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Hyperledger Fabric Operator for Kubernetes
+parent: Labs
+---
+# Lab Name
+Hyperledger Fabric Operator for Kubernetes
+
+# Short Description
+Manage Hyperledger Fabric components at scale with a Kubernetes Operator.
+
+# Scope of Lab
+I think there's a gap to deploy a Hyperledger Fabric network in Kubernetes, this project automates to automate the deployment.
```

Comment:
[nitpick] The words such as `I think`, `this project`, `automates to automate` can be better framed. For example

```suggestion
There is a gap in the Hyperledger Fabric network deployment on Kubernetes. The proposal is to automate the deployment using the concept of Kubernetes Operators.
```

Reply:
Added